lib/vdsm/storage/volume.py

```diff
@@ -1381,11 +1381,11 @@ def extendSize(self, new_capacity):
             raise se.VolumeNonWritable(self.volUUID)
 
         volFormat = self.getFormat()
```

Comment:
Looks good but please rewrite the commit message to plain English, I cannot parse it.

Reply:
Not sure which part do you mean?

Reply:
> "Relax assumption that cow volumes do now need to be extended to only those that are sparse."

I cannot parse this at all.

> "Preallocated volumes should be extended to avoid pausing VMs when it reaches its truesize, while
having bigger capacity."

This looks a bit better and I can probably guess the meaning but it's still not easy to understand. I understand it as: "Preallocated cow volumes should be extended too. Otherwise when a preallocated volume has a bigger capacity than is its `truesize` then the VM will pause once `truesize` is reached, which is not desirable."